app/src/main/java/com/example/hulk/mtindo/Store.java

```diff
@@ -0,0 +1,28 @@
+package com.example.hulk.mtindo;
+
+/**
+ * Created by hulk on 12/10/15.
+ */
+public class Store {
+    String UserName,Telephone,StoreName,Description;
```

Comment:
Please adhere to android coding standards for variable names etc : read https://source.android.com/source/code-style.html#follow-field-naming-conventions